models/models.go

```diff
@@ -155,7 +155,7 @@ func LoadConfigs() {
 		DbCfg.Passwd = sec.Key("PASSWD").String()
 	}
 	DbCfg.SSLMode = sec.Key("SSL_MODE").MustString("disable")
-	DbCfg.Path = sec.Key("PATH").MustString("data/gitea.db")
+	DbCfg.Path = sec.Key("PATH").MustString(path.Join(setting.AppDataPath, "gitea.db"))
```

Comment:
I'm going to put a `breaking` label on the PR due to this line, because it changes previous behaviour. So that it'll show up in the changelog as breaking as a notice to users that they may need to move the DB file to the appropriate location.

I am in full support of this change, as it will provide overall better experience (as I said in chat: sometimes breaking changes are needed, although we should be aware of when they are made, and alert users who may be affected)

Reply:
Sounds good. Since this PR has become much more about docs, do you want me to move that change to another PR since it's the only real "code" change?

Reply:
That is a good idea. I will remove the breaking label from this one, and add it to the new one.